projects/storefrontstyles/scss/theme/high-contrast/high-contrast.scss

```diff
@@ -0,0 +1,258 @@
+.high-contrast-dark,
+.high-contrast-light {
+  color: var(--cx-color-text);
+  background-color: var(--cx-color-background);
+
+  cx-site-context-selector,
+  cx-theme-switcher {
+    select {
+      background-color: var(--cx-color-background);
+      color: var(--cx-color-text);
+    }
+    cx-icon {
+      color: var(--cx-color-text);
+    }
+  }
+
+  cx-page-layout {
+    background-color: var(--cx-color-background) !important;
```

Comment:
Do we really have to put so many `!important`.
I guess this way we'll make customers' life much harder to extend those styles.

Reply:
I agree, let's use specificity as per Styling Guidelines: https://wiki.one.int.sap/wiki/display/spar/UI+Styling+and+Accessibility+Onboarding

Reply:
removed all important.